lib/Providers/Rng/RNGException.php

```diff
@@ -2,10 +2,6 @@
 
 namespace RobThree\Auth\Providers\Rng;
 
-class RNGException extends \Exception
-{
-    function __construct($message = "", $code = 0, $exception = null)
-    {
-    	parent::__construct($message, $code, $exception);
-    }
-}
\ No newline at end of file
+use RobThree\Auth\TwoFactorAuthException;
+
+class RNGException extends TwoFactorAuthException {}
```

Comment:
I have changed this to `TwoFactorAuthException`, is that OK for you? I can do the other as well (I forgot it I see)